test/docs/testplans/test_plan_template.md

```diff
@@ -0,0 +1,94 @@
+#Test name
```

Comment:
please separate this template into a different pr

Reply:
This template was borrowed from [SONiC test plan template](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20Test%20Plan%20Template.md) and placed here for reference. I can leave it out if not deemed necessary.

Reply:
Just make a link to this external document instead of duplicating it.

Reply:
fixed as per recommendation to link it to external document.